security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/policies.tf

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+# Copyright (c) 2022, 2023, Oracle and/or its affiliates.  All rights reserved.
+# This software is dual-licensed to you under the Universal Permissive License
+# (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl.
+###############################################################################
+#
+# Author: Fabrizio Zarri
+#
+################################################################################
+
+
+#  Policies
+
+resource "oci_identity_policy" "FunctionPolicy" {
+  provider = oci.homeregion
+  depends_on     = [oci_identity_dynamic_group.FunctionsServiceDynamicGroup]
```

Comment:
Terraform naming convention is all lower case and words separated by '_'.

Reply:
I have update the name of all variables and references. Now are only present lower case and words separated by '_'.